frontend/src/wells/views/WellDetail.vue

```diff
@@ -673,7 +682,10 @@ export default {
     isUnpublished () {
       return !this.well.is_published
     },
-    ...mapGetters(['userRoles', 'config', 'well', 'wellLicence', 'storedWellId', 'codes'])
+    isAuthenticated() {
+      return this.$keycloak && this.$keycloak.authenticated
+    },
+    ...mapGetters(['userRoles', 'config', 'well', 'wellLicence', 'storedWellId', 'codes', 'keycloak'])
```

Comment:
It looks like we are already using "userRoles" here so if we can use these instead, we save ourselves adding the keycloak code.

Reply:
Updated